Week3/README/READEME.md

```diff
@@ -0,0 +1,56 @@
+<h1>Run and Test the project:</h1>
+
+<h3> Clone repository:</h3>
+
+`https://github.com/ShyarK/databases.git`
+
+`npm i`
+
+`npm start`
+
+<h3> To get all users:</h3>
+
+`http://localhost:3000/users`
+
+<h3> To get all todolist:</h3>
+
+`http://localhost:3000/todolist`
+
+<h3> To get all todo items:</h3>
+
+`http://localhost:3000/todoitems`
+
+<h3> To get one user with the given id:</h3>
+
+`http://localhost:3000/user/7`
+
+<h3> To get one todoList with the given id:</h3>
+
+`http://localhost:3000/todolist/2`
+
+<h3> To get one item with the given id:</h3>
+
+`http://localhost:3000/todoitem/2`
+
+<h3> The todolist object:</h3>
+
+`{ "todolist_id": 2, "Name": "having dinner", "tag": "fun", "reminder": "2019-12-19T23:00:00.000Z", "user_id": 7 }`
+  
+
+
+  <h3> The item object:</h3> 
+  
+  ``{
+       "item_id": 2,
+       "description": "going to hyf amsterdam",
+       "todo_completed": "true",
+       "todolist_id": 2
+  }``
+   
+ <h3>The users object:</h3>
+ 
+ ``{
+        "user_id": 7,
```

Comment:
It's a capital U here otherwise it's not working

Reply:
I don't know if I get you right! You mean I need to change it in MySQL or in README.md file?
because I have it like thins:
```user_id` int(11) NOT NULL AUTO_INCREMENT``

Reply:
This is not sql here it's read the json keys.
The json keys are case sensitive and when you read the key later you read with User_id.
 > const { Name, User_id, Email } = req.body;
So either you change the readme or the code but it has to be consistent :)